studio/components/ui/Charts/BarChart/BarChart.types.ts

```diff
@@ -0,0 +1,34 @@
+import { CategoricalChartState } from "recharts/types/chart/generateCategoricalChart"
+
+export type  BarChartProps = {
+  data?: DataPoint[],
+  attribute: string,
+  yAxisLimit?: number | string,
+  format?: string,
+  highlightedValue?: number | string,
+  customDateFormat?: string,
+  displayDateInUtc?: boolean ,
+  label: string,
+  onBarClick?: (v: CategoricalChartState) => void,
+  minimalHeader?: boolean,
+  chartSize?: string,
+  className?: string,
+  noDataTitle?: string,
+  noDataMessage?: string,
+}
+
+export type HeaderType = {
```

Comment:
the header takes many props that are shared with the bar chart directly. should extend and pick from the bar chart interface.